index.js

```diff
@@ -56,6 +58,19 @@ DuplexWrapper.prototype._write = function _write(input, encoding, done) {
   this._writable.write(input, encoding, done);
 };
 
-DuplexWrapper.prototype._read = function _read(n) {
-  this._readable.resume();
+DuplexWrapper.prototype._read = function _read() {
+  if (this._shouldRead) return;
```

Comment:
Yeah, I was just looking at this because I couldn't get the module-deps tests to pass